backend/routes/usernameRoute.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
```

Comment:
since this subroute is for change username functionality it would make more sense to change the var name to changeUsernameRouter

Reply:
updated